airflow/contrib/hooks/gcp_cloud_build_hook.py

```diff
@@ -19,8 +19,8 @@
 """Hook for Google Cloud Build service"""
 
 import time
-
-from googleapiclient.discovery import build
+from typing import Optional
+from googleapiclient.discovery import build, Resource
```

Comment:
```suggestion
from googleapiclient.discovery import build, Resource
```
I am currently working on annotations for gcp_*_hook.py. 
https://github.com/apache/airflow/pull/5634
Can you delete changes from this PR to avoid conflicts? I will be happy if you do a review of my changes.

Reply:
Sure, I will wait for your PR to be merged first.

Reply:
I have removed my annotations for all the gcp hooks.